static/flow/views/landing-page-view.js

```diff
@@ -191,9 +220,12 @@ var LandingPageView = function(options) {
     }
     //
     // new program
-    //	
+    //    
```

Comment:
Not sure how I feel about these 3-line comments either honestly...if there's a large block of code doing a single thing, probably better to pull it into a function

Reply:
See above comment.  These 3 line comments were all over in the original code base.  I've gotten used to them, but they can be intrusive when first looking at it.